protobuf/flow/executiondata/executiondata.proto

```diff
@@ -63,6 +63,55 @@ service ExecutionDataAPI {
   rpc SubscribeEvents(SubscribeEventsRequest)
```

Comment:
let's add a deprecation notice to this rpc as well

Reply:
[@d01f7f0](https://github.com/onflow/flow/pull/1441/commits/d01f7f0cf505016e61fdfdf10cf2b49f83fb8eac)